### PR TITLE
Week 6_ 복습과제_최지우

### PR DESCRIPTION
**Pull request by chyu02**

### CODE REVIEW

**Comment by sudingoh** [comment]
지우님 안녕하세요!
과제 피드백이 늦어진 점 죄송합니다 ㅠㅠ 

1.6
HR['Attrition'] = HR.Attrition.astype("category").cat.codes
이 코드로도 해결 가능하니까 한번 확인해주시면 좋을 것 같습니다!

1.8 인코딩 할때 for문을 사용하셔도 좋을 것 같습니다! 
label = LabelEncoder()
for column in ['BusinessTravel','Department','EducationField','Gender','JobRole','MaritalStatus','OverTime']:
    X[column] = label.fit_transform(X[column])

과제 하시느라 정말 수고 많으셨습니다! 
즐거운 방학 보내세요~~!
